app/src/main/java/org/fossasia/openevent/general/auth/ProfileFragment.kt

```diff
@@ -63,11 +62,11 @@ class ProfileFragment : Fragment() {
 
         profileFragmentViewModel.user.observe(this, Observer {
             it?.let {
-                rootView.name.text = "${it.firstName.nullToEmpty()} ${it.lastName.nullToEmpty()}"
+                val userName = "${it.firstName.nullToEmpty()} ${it.lastName.nullToEmpty()}"
+                rootView.name.text = userName
                 rootView.email.text = it.email
                 emailSettings = it.email
 
-
```

Comment:
Why this change?

Reply:
Extra feed. Should I recover it? 

> Why this change?

Extra feed. Should I recover it?

Reply:
Yes